packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -244,10 +244,7 @@ export async function copyProjectTemplateAndReplace(
       id: 'ReactNative.Hermes.Windows',
       version: '0.6.3',
       hasProps: false,
-      // This does have build targets, but they're included automatically by
-      // Microsoft.ReactNative and don't need to be imported in the user
-      // project
-      hasTargets: false,
+      hasTargets: true,
```

Comment:
Drive by fix here. Added this change late in the last PR, and it meant that we no longer copied Hermes.dll to the appx (only kept it internal to the build). Seems like there's no way around needing to alter the imports in the template project without changing build logic further.